app/protected/route.js

```diff
@@ -3,13 +3,12 @@ import Ember from 'ember';
 export default Ember.Route.extend({
 
   model(){
-
       return Ember.RSVP.hash({
         jobs: this.store.query('job', {
-          orderBy: "isActive",
-          equalTo: true
+          orderBy: "assignedTo",
```

Comment:
The issue here is that now if a job gets disabled(soft deleted), it will still show.